### DISCUSSION TITLE
Add iOS platform-specifics for translucency to Shell

### DISCUSSION THREAD

**Discussion by warpedgeoid** [dotnet/maui#22403] [Ideas]
### Description

It is possible to set translucency effects on the navigation bar and tab bar when using iOS platform-specifics targeting NavigationPage and TabbedPage, respectively, however, it is not possible to do the same when using Shell.

I am requesting that SetTranslucencyMode() and GetTranslucencyMode() APIs are added that target Shell.

### Public API Changes

Something similar to the following:

```
// Get the current shell.
var shell = Shell.Current;
// Set the translucency mode.
Shell.On<iOS>().SetTranslucencyMode(shell, TranslucencyMode.Translucent);
// Get the translucency mode.
var mode = Shell.On<iOS>().GetTranslucencyMode(shell);
```

### Intended Use-Case

This feature would make it possible for apps using Shell to achieve the same visual styling as apps using NavigationPage and TabbedPage directly.

I would like to make our app visually consistent with our previous native iOS app while still being able to take advantage of the great Shell provided by Maui.

**Comment by PureWeen** [comment]
@warpedgeoid I know you logged this some time ago :-) 

What's the setup of your NavigationPage where you are taking advantage of `Translucency` ?

With Shell, if we just set Translucency to true when you have a transparent NavigationBar would that be enough?

**Reply by warpedgeoid** [reply]
Hey @PureWeen. Unfortunately, I'm no longer working on the codebase where I encountered this issue.
